src/app.rs

```diff
@@ -362,7 +361,7 @@ where
         self.block = block;
     }
 
-    // this let's use use "next block" steps that add eg. one height and 5 seconds
+    /// Advances the current block to "next block" steps and adds one height and 5 seconds.
```

Comment:
I think this comment is incorrect. This method updates the block info by applying some closure.

This comment should refer to the `next_block` function.

Reply:
Right, fixed.